solutions/project/spoke-project-unclass/hostvpcproject.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
```

Comment:
I suggest you move this resource inside the project.yaml file

Reply:
done